src/devices/Ads1115/Ads1115.cs

```diff
@@ -62,37 +140,106 @@ public DataRate DataRate
         }
 
         /// <summary>
-        /// Initialize a new Ads1115 device connected through I2C
+        /// ADS1115 operation mode.
+        /// When set to <see cref="DeviceMode.Continuous"/> the chip continously measures the input and the values can be read directly.
+        /// If set to <see cref="DeviceMode.PowerDown"/> the chip enters idle mode after each conversion and
+        /// a new value will be requested each time a read request is performed. This is the recommended setting when frequently
+        /// swapping between input channels, because a change of the channel requires a new conversion anyway.
         /// </summary>
-        /// <param name="i2cDevice">The I2C device used for communication.</param>
-        /// <param name="inputMultiplexer">Input Multiplexer</param>
-        /// <param name="measuringRange">Programmable Gain Amplifier</param>
-        /// <param name="dataRate">Data Rate</param>
-        public Ads1115(I2cDevice i2cDevice, InputMultiplexer inputMultiplexer = InputMultiplexer.AIN0, MeasuringRange measuringRange = MeasuringRange.FS4096, DataRate dataRate = DataRate.SPS128)
+        public DeviceMode DeviceMode
         {
-            _i2cDevice = i2cDevice;
-            _inputMultiplexer = inputMultiplexer;
-            _measuringRange = measuringRange;
-            _dataRate = dataRate;
+            get
+            {
+                return _deviceMode;
+            }
+            set
+            {
+                _deviceMode = value;
+                SetConfig();
+            }
+        }
 
-            SetConfig();
+        /// <summary>
+        /// Comparator mode.
+        /// Only relevant if the comparator trigger event is set up and is changed by <see cref="EnableComparator(short, short, ComparatorMode, ComparatorQueue)"/>.
+        /// </summary>
+        public ComparatorMode ComparatorMode
+        {
+            get;
+            private set;
```

Comment:
No need to call SetConfig() here?

Reply:
No, the value can only be changed by calling EnableComparator, where the configuration is applied directly.